examples/java/spring-boot/src/test/java/uk/gov/api/springboot/models/metadata/DataTest.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.api.springboot.models.metadata;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.json.JsonTest;
+import org.springframework.boot.test.json.JacksonTester;
+import org.springframework.boot.test.json.JsonContent;
+
+@JsonTest
+class DataTest {
+
+  @Autowired private JacksonTester<Data> jacksonTester;
```

Comment:
```suggestion
  @Autowired
  private JacksonTester<Data> jacksonTester;
```

Reply:
this is something SpotlessCheck does, do we want to (/can we) override it?

Reply:
Ah sorry, if this is Spotless doing it it's all good - my preference is that it's split, but the whole point of using Spotless with Google Code Style is to make sure my personal preferences go out the window :grin: